website/markdown/docs/dependencies/dependencies.mdx

```diff
@@ -38,7 +38,7 @@ let dependencies = Dependencies(
             .github(path: "Alamofire/Alamofire", requirement: .exact("5.0.4"))
         ],
         platforms: [.iOS, .macOS],
-        useXCFrameworks: true
+        options: [.useXCFrameworks]
```

Comment:
shouldn't the options as a whole be documented here? Eg `.noUseBinaries` options is not documented on the website right now